testing/lib/workarea/test_case.rb

```diff
@@ -208,8 +208,34 @@ def reset_encryption_key
       end
     end
 
+    module Setup
```

Comment:
If anyone can think of a word to represent both setup and teardown, I'm into combining these into a single module.

Reply:
`TestCase::Environment` doesn't necessarily represent the two verbs you mentioned, but IMO it's what this module does: sets up and tears down the _environment_ used to run tests. You could also call it `TestCase::Harness` in relation to the alternative definition here: https://en.wikipedia.org/wiki/Test_harness

Reply:
also thank you for doing this, i remember going down a rabbit hole trying to add mailer tests once and i realized i had to do a lot of copy pasta